### PR TITLE
docs: added back docs for release 2.3

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

sync master with v2.3

### CODE REVIEW

**Review by LiteSun — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,13 +19,38 @@
...
 - [1.0.0](#100)
 
+# 2.3.0
+
+### Core
+
+* Support to use absolute path in conf.WebDir. [#1055](https://github.com/apache/apisix-dashboard/pull/1055)
+* Support to search route by label. [#1061](https://github.com/apache/apisix-dashboard/pull/1061)
+* Support server info preview. [#958](https://github.com/apache/apisix-dashboard/pull/958) [#949](https://github.com/apache/apisix-dashboard/pull/949)
+* Support search route by labels. [#1061](https://github.com/apache/apisix-dashboard/pull/1061)
```
Comment (liuxiran) [inline CHANGELOG.md:37]: duplicate log

```suggestion
* Support to search route by label. [#1061](https://github.com/apache/apisix-dashboard/pull/1061)
* Support server info preview. [#958](https://github.com/apache/apisix-dashboard/pull/958) [#949](https://github.com/apache/apisix-dashboard/pull/949)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,13 +19,37 @@
...
+* Use Cobra as the cli scafford. [#773](https://github.com/apache/apisix-dashboard/pull/773)
+* Remove Lua dependency. [#1083](https://github.com/apache/apisix-dashboard/pull/1083)
+* Improve E2E testcases for the backend. [#1012](https://github.com/apache/apisix-dashboard/pull/1012), [#1123](https://github.com/apache/apisix-dashboard/pull/1123)
+* Improve E2E testcases for the frontend. [#1074](https://github.com/apache/apisix-dashboard/pull/1074)
+* Improve online debug. [#979](https://github.com/apache/apisix-dashboard/pull/979)
+* Improve Route publish/offline. [#1081](https://github.com/apache/apisix-dashboard/pull/1081) [#991](https://github.com/apache/apisix-dashboard/pull/991)
+* Improve plugin module for the frontend. [#1047](https://github.com/apache/apisix-dashboard/pull/1047) [#978](https://github.com/apache/apisix-dashboard/pull/978)
+* Fix error occurred when enable or disable existing SSL [#1064](https://github.com/apache/apisix-dashboard/pull/1064)
+* Fix the problem that route created by Admin API (without ID) cannot be shown in Manager API. [#1063](https://github.com/apache/apisix-dashboard/pull/1063)
```
Comment (starsz) [inline CHANGELOG.md:53]: Hi, I think we need to split them into `core`, `bugfix`, and `test case` like before.

**Comment by imjoey** [comment]
@juzhiyuan since there is only 3 commits about the changelog and deploy guide. So, just FYI, could we change the title of this PR to: `feat(doc): Add back docs for release 2.3` ? IMHO, that is more precise.  😄

**Comment by juzhiyuan** [comment]
more clear!

**Review by tokers — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (juzhiyuan) [reply CHANGELOG.md:53]: Should file another PR to improve all changelogs both for master and v2.3

**Pull request merged**